astropy/coordinates/tests/test_solar_system.py

```diff
@@ -26,7 +26,7 @@
 
 separation_tolerance_planets = 5*u.arcsec
 separation_tolerance_moon = 5*u.arcsec
-distance_tolerance = 20  # km
+distance_tolerance = 20*u.km  # km
```

Comment:
If you make another round of changes, delete `# km` here (and maybe add spaces around `*` and in the lines above.